server/db/migrations/20171020144702_initial_schema.js

```diff
@@ -0,0 +1,41 @@
+exports.up = knex =>
+	knex.schema
+		.raw('create extension if not exists "uuid-ossp"')
+		.createTable('flair', table => {
+			table.increments('id').primary().notNullable()
+			table.string('name', 20).notNullable()
+			table.string('description', 100)
+			table.text('filename').notNullable()
+		})
+		.createTable('ball', table => {
```

Comment:
what is the point of this table? for custom balls?

Reply:
Yep, that's the idea. That can be removed for the time being though as it isn't fleshed out and certainly won't be in MVP.